beta/3.0.0/README.md

```diff
@@ -0,0 +1,447 @@
+- `Content-Type: application/json`
+
+#### **Request Body:**
+
+- **`grant_type`**: The type of OAuth2 grant.
+  - **Required**: Yes
+  - **Value**: Must be `"client_credentials"`
+  - **Nullable**: No
+
+#### **Example cURL:**
+
+```bash
+curl --request POST 'https://api.pulsifi.me/partner/oauth2/token' \
+  --header 'Accept: application/json' \
+  --header 'Authorization: Basic YOUR-ENCODED-CLIENT-ID-CLIENT-SECRET' \
+  --header 'Content-Type: application/json' \
+  --data-raw '{"grant_type": "client_credentials"}'
+```
+
+#### **Response Example:**
+
+```json
+{
+  "access_token": "string",
+  "scope": "string",
+  "expires_in": "string",
+  "token_type": "bearer"
+}
+```
+
+#### **Response Body:**
+
+- **`access_token`**: The token that must be included in the `Authorization` header of all subsequent API requests.
+
+  - **Type**: String
+  - **Nullable**: No
+
+- **`scope`**: The permissions granted to the access token, defining what actions or resources it can access.
+
+  - **Type**: String
+  - **Nullable**: No
+
+- **`expires_in`**: The lifetime of the access token in seconds.
+
+  - **Type**: String
+  - **Nullable**: No
+
+- **`token_type`**: The type of the token, usually `bearer`, indicating how the token should be included in API requests.
+
+  - **Type**: String
+  - **Nullable**: No
+
+---
+
+## **Endpoints**
+
+### **1. Create Candidate Invitation**
+
+Creates a new candidate invitation in the Pulsifi system.
+
+- **Endpoint:** `POST /partner/v1.0/standard/candidates`
+- **Request URL:** `https://api.pulsifi.me/partner/v1.0/standard/candidates`
+
+#### **Headers:**
+
+- `Accept: application/json`
+- `Authorization: Bearer <access_token>`
+- `Content-Type: application/json`
+
+#### **Request Body Parameters:**
+
+- **`job_id`**: The unique identifier for the job in the Pulsifi system. Refer to the [FAQ](#faq) below for details on how to obtain this ID.
+
+  - **Required**: Yes
+  - **Type**: String (UUID)
+  - **Max Length**: 36 characters
+  - **Nullable**: No
+
+- **`ext_reference_id`**: External job application ID from the Applicant Tracking System (ATS).
+
+  - **Required**: Yes
+  - **Type**: String
+  - **Max Length**: 50 characters
+  - **Nullable**: No
+
+- **`is_anonymous_candidate`**: Indicates whether the candidate's details should be anonymous.
+
+  - **Required**: Yes
+  - **Type**: Boolean
+  - **Nullable**: Yes
+  - **Behavior**: If set to `true`, the `email`, `first_name`, and `last_name` fields become optional.
+
+- **`email`**: The candidate's email address.
+
+  - **Required**: No (if `is_anonymous_candidate` is `true`)
+  - **Type**: String
+  - **Max Length**: 255 characters
+  - **Nullable**: Yes
+
+- **`first_name`**: The candidate's first name.
+
+  - **Required**: No (if `is_anonymous_candidate` is `true`)
+  - **Type**: String
+  - **Max Length**: 255 characters
+  - **Nullable**: Yes
+
+- **`last_name`**: The candidate's last name.
+
+  - **Required**: No (if `is_anonymous_candidate` is `true`)
```

Comment:
Can make this as not required and default to false , move it to last field 

See if we can offload anonymous mode to client 

Like Testlify or willo  also wont have this 

It up us to send in a fake email / name or not

Reply:
Same behavior as deadline right? Not required and default to 3 months

Reply:
Aite make sense.

Make email , first_name, last_name (optional)

We default them with fake email/name on our side if not provide.
Remove is_anonymous key.

Yeah deadline will be default 3 months if not provided.

Reply:
as per discussion
- stick with is_anonymous , not required,  default false **Don sell this in api doc in future